### PR TITLE
Update faq.yml

### PR DESCRIPTION
**Pull request by dagdome**
New FAQ ASL AO

## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [ ] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Comment by dagdome** [comment]
please @bfabio @denismarini

**Review by denismarini — APPROVED** [review]

**Comment by denismarini** [comment]
It's done @dagdome!

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -563,6 +566,62 @@ questions:
...
+      - title: Un Comune è obbligato ad adottare la stessa modalità di migrazione per tutti i servizi?
+        content: No, l’Ente potrà selezionare - singolarmente per ognuno dei servizi oggetto della migrazione - il modello di migrazione più adatto da presentare all’interno del piano di migrazione.
+        tag:
+          - Asl/AO
+      - title: Cosa contiene il piano di migrazione?
+        content: Il piano di migrazione dei Soggetti Attuatori conterrà al suo interno tutta la lista di servizi che verranno migrati verso il cloud qualificato.
+        tag:
+          - Asl/AO
+      - title: Qual è la destinazione prevista per i servizi inseriti nel piano di migrazione?
+        content:La destinazione dei servizi inseriti nel piano di migrazione previsto dall’avviso è:<br>- Per i servizi ordinari:Public Cloud Qualificato, Infrastruttura della PA idonea o PSN; </br> <br>- Per i servizi critici": Polo Strategico Nazionale (PSN).</br>
```
Comment (bfabio) [inline contents/faq-page/faq.yml:602]: Questo è sbagliat

Comment (dagdome) [reply contents/faq-page/faq.yml:602]: Ciao Fabio , mi aiuti a correggerlo ? grazie

Comment (denismarini) [reply contents/faq-page/faq.yml:602]: Grazie @bfabio ora ci guardo insieme a @dagdome

**Pull request merged**